xml/bp_troubleshooting_tips.xml

```diff
@@ -1228,21 +1228,22 @@ ID WEIGHT TYPE NAME    UP/DOWN REWEIGHT PRIMARY-AFFINITY
     <para>
      Refresh the &spillar;:
     </para>
-    <screen>&prompt.smaster;salt '*' saltutil.pillar_refresh</screen>
+<screen>&prompt.smaster;salt '*' saltutil.pillar_refresh</screen>
    </listitem>
    <listitem>
     <para>
      Verify the changed value:
     </para>
-    <screen>&prompt.smaster;salt '*' pillar.items</screen>
+<screen>&prompt.smaster;salt '*' pillar.items</screen>
    </listitem>
    <listitem>
     <para>
      Apply the new setting:
     </para>
-    <screen>&prompt.smaster;salt '*' state.apply ceph.time</screen>
+<screen>&prompt.smaster;salt '*' state.apply ceph.time</screen>
```

Comment:
That's not correct anymore. @ricardoasmarques Is there any 'troubleshooty' way to add timeservers via salt that is worth adding here?